scripts/generate-integration-pages.ts

```diff
@@ -178,6 +179,41 @@ function absoluteLinks({ base }: { base: string }) {
 	};
 }
 
+/** Remark plugin to replace GitHub . */
+function replaceAsides() {
+	return function transform(tree: Root) {
+		visit(tree, 'blockquote', (node) => {
+			const openingParagraph = node.children[0];
+			const [firstChild, trailingText] = openingParagraph.children;
+
+			// check for **Note:** or **Warning:** at the beginning of the first paragraph
+			if (firstChild.type === 'strong' && /Note|Warning/.test(firstChild.children[0].value)) {
```

Comment:
Style nit you can skip if you prefer: this could be an opportunity for an early return:

```js
if (
	firstChild.type !== 'strong') ||
	!/Note|Warning/.test(firstChild.children[0].value)
) {
	return;
}
```

That would avoid nesting everything that follows inside the `if` statement. But it’s fine as-is too.

Reply:
I think I saw early returns as common practice in remark plugins. I will add this 😇